test/common/http/header_map_impl_test.cc

```diff
@@ -692,6 +692,15 @@ TEST(HeaderMapImplTest, TestAppendHeader) {
     HeaderMapImpl::appendToHeader(value3, "");
     EXPECT_EQ(value3, "empty");
   }
+  // Regression test for appending to an empty string with empty, then setting integer.
+  {
+    const std::string empty;
+    HeaderString value4(empty);
+    HeaderMapImpl::appendToHeader(value4, " ");
```

Comment:
This doesn't match the comment, since `" "` isn't an empty string.

Reply:
True; the original fuzz case looked like an empty string, but I'm thinking it might have been strange Unicode.

Reply:
Oh, that's terrifying.